mappings/net/minecraft/client/gui/screen/ingame/InventoryScreen.mapping

```diff
@@ -15,3 +15,10 @@ CLASS net/minecraft/unmapped/C_lkskxcwz net/minecraft/client/gui/screen/ingame/I
 		ARG 2 y
 		ARG 3 size
 		ARG 6 entity
+	METHOD m_yyefzvem drawEntityWithLookPosition (Lnet/minecraft/unmapped/C_cnszsxvd;IIIFFLnet/minecraft/unmapped/C_usxaxydn;)V
```

Comment:
this is a tad long, I think with the parameters `lookX` and `lookY` we don't have to clarify `WithLookPosition`